api/Dockerfile

```diff
@@ -24,7 +24,16 @@ WORKDIR /app
 RUN chown -R nonrootuser /app
 COPY pyproject.toml poetry.lock ./
 RUN poetry install
-COPY . .
+COPY ./__pycache__ ./__pycache__
```

Comment:
These are generated cache files that don't need to be added. They can even contain secrets